examples/cpp/pyperf/PyPerfCollapsedPrinter.cc

```diff
@@ -78,22 +78,22 @@ void PyPerfCollapsedPrinter::processSamples(
     std::fprintf(output_file, "%s-%d/%d", sample.comm.c_str(), sample.pid, sample.tid);
     switch (sample.stackStatus) {
     case STACK_STATUS_TRUNCATED:
-      std::fprintf(output_file, ";%s", kIncompleteStack.c_str());
+      std::fprintf(output_file, ";%s_[p]", kIncompleteStack.c_str());
       truncatedStack++;
       break;
     case STACK_STATUS_ERROR:
-      std::fprintf(output_file, ";%s", kErrorStack.c_str());
+      std::fprintf(output_file, ";%s_[p]", kErrorStack.c_str());
       errors++;
       break;
     default:
       for (auto it = sample.pyStackIds.crbegin(); it != sample.pyStackIds.crend(); ++it) {
         const auto stackId = *it;
         auto symbIt = symbols.find(stackId);
         if (symbIt != symbols.end()) {
-          std::fprintf(output_file, ";%s", symbIt->second.c_str());
+          std::fprintf(output_file, ";%s_[p]", symbIt->second.c_str());
           frames++;
         } else {
-          std::fprintf(output_file, ";%s", kLostSymbol.c_str());
+          std::fprintf(output_file, ";%s_[p]", kLostSymbol.c_str());
```

Comment:
Don't add the suffix for the error messages (lost, error, incomplete), only for the python frames.

Reply:
Not sure. It's a UI question. I think it's best if all Python stacks are of the same color, no?

Reply:
It's a question of frames, not stacks. The Python frames will be Python colored, the error frames should be error colored.

Reply:
Thing is, they won't be error colored, they'll just be "red" because that's the default.

We could have specific identification for "error" frames, then make sure those are colored that way. This can also mark `[unknown]` perf frames... would be nicer indeed.

K can remove the annotation from those frames now.

Reply:
Sounds good to me. @d3dave ?

I do think it's nicer if colored different (but different as in "not Python & not Other")

Reply:
Fine by me.